Week3/Company_create_insert_fk.sql

```diff
@@ -0,0 +1,104 @@
+CREATE DATABASE IF NOT EXISTS Company;
+USE Company;
+
+CREATE TABLE IF NOT EXISTS Employee (
+  eID INT(4) NOT NULL,
+  eName VARCHAR(45) NOT NULL,
+  initials VARCHAR(10) NULL DEFAULT NULL,
+  dateOfBirth DATE NOT NULL,
+  position VARCHAR(20) NOT NULL,
+  salary DECIMAL(7,2) NOT NULL,
+  room VARCHAR(10) NOT NULL,
+  bossNr INT(4) NULL DEFAULT NULL,
+  dID INT(2) NOT NULL,
+  PRIMARY KEY (eID));
+
+CREATE TABLE IF NOT EXISTS Location (
+  code VARCHAR(5) NOT NULL,
+  description VARCHAR(45) NOT NULL,
+  country VARCHAR(45) NOT NULL,
+  mgrID INT(4) NULL DEFAULT NULL,
+  PRIMARY KEY (code));
+
+CREATE TABLE IF NOT EXISTS Department (
+  dID INT(2) NOT NULL,
+  dName VARCHAR(25) NOT NULL,
+  LocationCode VARCHAR(5) NOT NULL,
+  secretaryID INT(4) NULL DEFAULT NULL,
+  PRIMARY KEY (dID));
+
+CREATE TABLE IF NOT EXISTS SalaryScale (
+  scale INT(2) NOT NULL,
+  lowerLimit DECIMAL(7,0) NOT NULL,
+  upperLimit DECIMAL(7,0) NOT NULL,
+  PRIMARY KEY (scale));
+
+ALTER TABLE Employee
+  ADD CONSTRAINT Employee_has_boss_fk
+    FOREIGN KEY (bossNr) REFERENCES Employee (eID),
```

Comment:
Ah, I see - you added the constraints down here.
It would be more clear to make them part of the CREATE TABLE.